packages/@react-aria/interactions/src/useFocusVisible.ts

```diff
@@ -196,7 +196,7 @@ const tearDownWindowFocusTracking = (element, loadListener?: () => void) => {
  * Adds a window (ie iframe) to the list of windows that are being tracked for focus visible.
```

Comment:
Super-nit, since this will be in the docs.

```suggestion
 * Adds a window (i.e. iframe) to the list of windows that are being tracked for focus visible.
```